plugins/derlord/OSS.md

```diff
@@ -36,71 +41,66 @@ So feel free to do whatever you want!
-    { x: 25.980966567993164, y: -1345.6417236328125, z: 28.497024536132812 } as alt.Vector3, // Index 0
-];
-
-// Used to give all the shops are different blip, just copy the line as often as you need it.
-export const buyerBlips = [
-    { sprite: 59, color: 2, text: '24/7 Shop', scale: 1 }, // Index 0
-];
-
-// In OSS each Shop can hold individual items or you can add the same list to a few more stores over and over again.
-// Default List for all 24/7 & LTD Stores.
-const shopList = [
-    { name: 'Bread', dbName: 'bread', price: 75, image: 'crate' },
-    { name: 'Hotdog', dbName: 'Shophotdog', price: 375, image: 'crate' },
-    { name: 'Waterbottle', dbName: 'Shopwater', price: 250, image: 'crate' },
-    { name: 'Cola', dbName: 'Shopcola', price: 250, image: 'crate'},
-    { name: 'Energy Drink', dbName: 'Shopenergy', price: 300, image: 'crate'}
-];
+- import './OSS/index';
 
-export const buyLists = [
-    shopList,
-]; // ADD YOUR LISTS HERE!
+src-webviews/pages/components.ts ->
+- import ShopUI from './shopUI/ShopUI.vue';
+- ShopUI: shallowRef(ShopUI)
 ```
-
-# Setup (Drugsellers)
+# Setup (Shops / TYPE )
 ```typescript
-import * as alt from 'alt-server';
-
-// Shops are based on the index of arrays so -> First index -> First sellList
-export const SELLERS: alt.Vector3[] = [
-    { x: 1163.400634765625, y: -323.938232421875, z: 68.20509338378906 } as alt.Vector3 // SELLER (SHOP-POS) INDEX -> 0
-    // { Add stuff yourself } Just another seller, INDEX 1 -> The secoond list will get into this position.
-];
-
-// Used to give all the shops are different blip, just copy the line as often as you need it.
-export const sellersBlips = [
-    { sprite: 52, color: 1, text: 'Seller Example - Index 0', scale: 1 }, // Index 0
+export const coreShopLocations: IShopLocation[] = [
+    { x: 25.980966567993164, y: -1345.6417236328125, z: 28.497024536132812, isBlip: true } as IShopLocation, 
+    { x: 374.3475341796875, y: 328.112060546875, z: 102.56637573242188, isBlip: true } as IShopLocation, 
+    { x: -3041.32763671875, y: 585.155029296875, z: 6.908928871154785, isBlip: true } as IShopLocation, 
+    { x: -3243.743408203125, y: 1001.3903198242188, z: 11.830706596374512, isBlip: true } as IShopLocation, 
+    { x: 548.0447387695312, y: 2669.48876953125, z: 41.156490325927734, isBlip: true } as IShopLocation, 
+    { x: 1960.2322998046875, y: 3742.317138671875, z: 31.343746185302734, isBlip: true } as IShopLocation, 
+    { x: 1730.01171875, y: 6416.22021484375, z: 34.03722381591797, isBlip: true } as IShopLocation, 
+    { x: 2555.4609375, y: 382.1643371582031, z: 107.62295532226562, isBlip: true } as IShopLocation, 
 ];
 
-const burgerBuyer = [
-    { name: 'Burger', dbName: 'burger', price: 330, image: 'crate' }, // SellList INDEX -> 0
-    // { Add stuff yourself. } Second Items for SELLERS Index -> 0
-];
-const anotherExampleList = [
-    // { Fill me! } -> Index 1
+export const ShopRegistry: Array<IShop> = [
```

Comment:
Same here, where's the type `IShop` defined?

Reply:
Will be resolved soon. Thanks 👍